ui_test/POM_hiway_asg/pages/loging_using_google.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from POM_hiway_asg.pages.constant_var import LOGIN_WITH_GOOGLE_BTN, USER_NAME_IDENTIFIER, PASSWORD_IDENTIFIER, \
+    START_USING_HIWAY_BTN, TIMESHEET_TAB_BTN, VALID_USERNAME_LOGIN, VALID_PASSWORD_LOGIN
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+
+logging.basicConfig(filename='login_using_google_log.log',
+                    format='%(asctime)s: %(levelname)s %(message)s',
```

Comment:
extract all constants. also will you have a different log file for every code file?